layers/meta-resin-generic/recipes-support/resin-init/resin-init-flasher-board/resin-init-flasher-board

```diff
@@ -1,7 +1,33 @@
 #!/bin/sh
 
+INSTALL_UEFI=""
+INTERNAL_DEVICE_BOOT_PART_MOUNTPOINT=/tmp/internal_boot
+
+efibootmgr &>/dev/null ; INSTALL_UEFI=$?
+
 set -e
```

Comment:
Shouldn't we set -e at the start of the script?

Reply:
We are trying to capture the exit code of efibootmgr, if it fails we do not want the script to exit.